libs/npm/core/src/components/grid-table/GridTable.tsx

```diff
@@ -225,7 +225,7 @@ export const GridTable = <T extends object>({
             prepareRow(row);
             return (
               <div
-                className={activeId && activeId === (row.original as any).id ? 'active' : ''}
+                className={!!activeId && activeId === (row.original as any).id ? 'active' : ''}
```

Comment:
Why was this needed?

Reply:
For some reason on the time log table when no prop was provided it was still triggering.. but it had different behaviour on other pages (like ingestion)